src/current/_includes/releases/v24.1/v24.1.8.md

```diff
@@ -0,0 +1,73 @@
+## v24.1.8
+
+Release Date: December 12, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-8-security-updates">Security updates</h3>
+
+- All cluster settings that accept strings are now fully redacted when transmitted as part of our diagnostics telemetry. The transmitted payload includes a record of modified cluster settings and their values when they are not strings. If you previously applied the mitigations in [Technical Advisory 133479]({% link advisories/a133479.md %}), you can safely set the value of cluster setting `server.redact_sensitive_settings.enabled` to false and turn on diagnostic reporting via the `diagnostics.reporting.enabled` cluster setting without leaking sensitive cluster settings values. [#134016][#134016]
+
+<h3 id="v24-1-8-general-changes">General changes</h3>
+
+- COCKROACH_SKIP_ENABLING_DIAGNOSTIC_REPORTING is no longer mentioned in the `cockroach demo` command. [#134087][#134087]
+- Added `system.users` to the list of system tables that changefeeds protect with Protected Timestamps (PTS). This table is required for Change Data Capture Queries. [#134836][#134836]
+- Added changefeed support for the `mvcc_timestamp` option with the `avro` format. If both options are specified, the `avro` schema includes an `mvcc_timestamp` metadata field and emits the row's MVCC timestamp with the row data. [#136482][#136482]
+
+<h3 id="v24-1-8-operational-changes">Operational changes</h3>
+
+- The `goschedstats.always_use_short_sample_period.enabled` setting should be set to `true` for any production cluster, to prevent unnecessary queuing in admission control CPU queues. [#133583][#133583]
```

Comment:
I would remove the comma here, or flip the sentence (To prevent unnecessary...., the `goschedstats.always_use_short_sample_period.enabled setting)

Reply:
flipped sentence